key-manager.js

```diff
@@ -89,13 +90,12 @@ class KeyManager {
   }
 
   /**
-   * Decrypt an encrypted message (uses the first 24 bytes of the projectId as a nonce)
+   * Decrypt an encrypted message using the provided nonce parameter
    *
    * @param {Buffer} cyphertext
-   * @param {string} projectId
+   * @param {Buffer} nonce Buffer of length sodium.crypto_aead_xchacha20poly1305_ietf_NPUBBYTES bytes
```

Comment:
is it clearer to just say the exact length (24 bytes), assuming it doesn't change?